pkg/cloud/services/networking/securitygroups.go

```diff
@@ -26,10 +26,11 @@ import (
 )
 
 const (
-	secGroupPrefix     string = "k8s"
-	controlPlaneSuffix string = "controlplane"
-	workerSuffix       string = "worker"
-	bastionSuffix      string = "bastion"
+	SecGroupPrefix     string = "k8s"
+	ControlPlaneSuffix string = "controlplane"
+	WorkerSuffix       string = "worker"
+	BastionSuffix      string = "bastion"
+	NeutronLbaasSuffix string = "lbaas"
```

Comment:
An unexported constant is `secGroupPrefix` not `SecGroupPrefix`

https://github.com/golang/go/wiki/CodeReviewComments#mixed-caps

Reply:
SecGroupPrefix is used in https://github.com/zoltan0907/cluster-api-provider-openstack/blob/master/pkg/cloud/services/loadbalancer/loadbalancer.go#L181

Reply:
This is about naming convention.
Check out https://github.com/golang/go/wiki/CodeReviewComments#mixed-caps

Reply:
Sadly i don't understand what you want me todo. Should `SecGroupPrefix` transformed into `SECGroupPrefix` or shoud i add a comment to the export? I rewrote this peace of code and made a exported function to calculate the name of the securitygroup. Hopefully this is sufficient.

Reply:
NeutronLbaasSuffix is used in https://github.com/zoltan0907/cluster-api-provider-openstack/blob/master/pkg/cloud/services/loadbalancer/loadbalancer.go#L181. I can not find SecGroupPrefix in the line.